### PR TITLE
Addon: Helm

### PR DESCRIPTION
**Pull request by jakolehm**
Deploys [Helm](https://www.helm.sh) Tiller to `kube-system` namespace. Tiller is secured so that it listens only on localhost. In practice this means that only users with proper RBAC rights can access tiller via `helm` cli.

## Addon configuration

```yaml
addons:
  helm:
    enabled: true
```

## Helm CLI configuration

```
$ helm init -c
```

### CODE REVIEW

File: addons/helm/resources/deployment.yml.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        app: helm
+        name: tiller
+    spec:
+      containers:
+      - env:
+        - name: TILLER_NAMESPACE
+          value: kube-system
+        - name: TILLER_HISTORY_MAX
+          value: "10"
+        image: gcr.io/kubernetes-helm/tiller:v<%= version %>
```
Comment (nevalla) [inline addons/helm/resources/deployment.yml.erb]: where is this coming from?

Comment (jakolehm) [reply addons/helm/resources/deployment.yml.erb]: Do you mean version or?

Comment (nevalla) [reply addons/helm/resources/deployment.yml.erb]: yes, version.

Comment (jakolehm) [reply addons/helm/resources/deployment.yml.erb]: It's the addon version.

**Pull request merged**